SmartDeviceLinkTests/DevAPISpecs/SDLSoftButtonManagerSpec.m

```diff
@@ -94,10 +96,11 @@ - (void)sdl_displayCapabilityDidUpdate:(SDLSystemCapability *)systemCapability;
         softButtonCapabilities.longPressAvailable = @YES;
         softButtonCapabilities.shortPressAvailable = @YES;
 
-        SDLWindowCapability *windowCapability = [[SDLWindowCapability alloc] init];
-        windowCapability.softButtonCapabilities = @[softButtonCapabilities];
-        SDLDisplayCapability *displayCapability = [[SDLDisplayCapability alloc] initWithDisplayName:@"TEST" windowCapabilities:@[windowCapability] windowTypeSupported:nil];
-        [testManager sdl_displayCapabilityDidUpdate:[[SDLSystemCapability alloc] initWithDisplayCapabilities:@[displayCapability]]];
+        testWindowCapability = [[SDLWindowCapability alloc] init];
+        testWindowCapability.softButtonCapabilities = @[softButtonCapabilities];
+
+        OCMExpect([testSystemCapabilityManager defaultMainWindowCapability]).andReturn(testWindowCapability);
```

Comment:
Verify?

Reply:
The way the tests were originally written, you had to stub the `testWindowCapability` twice. Unfortunately, you can not stub the same method twice and return different values but you can with `OCMExpect. andReturn ()`. I rewrote the tests so the `hmiLevel` and `softButtonCapabilities` are set before each set of tests so stubs can be used since not all the tests are expected to call `defaultMainWindowCapability`.